source/input.cpp

```diff
@@ -2384,6 +2384,15 @@ void Input::Check(void)
 			{
 				ModuleBase::WARNING_QUIT("Input","not ready for linear_scaling method in lcao .");
 			}
+			else if (ks_solver == "cusolver")
+			{
+				
+#ifdef __MPI
+				GlobalV::ofs_warning << "Cusolver is now only available for LCAO." << std::endl;
```

Comment:
Does this PR supports MPI? If so, here should terminates the program same as the non-MPI part.

Reply:
Actually, for the easy coding, `cusolver` ks_solver in this PR support MPI only. So I edited here to warn if MPI is off.